From a8f29e2683e97557f0c8be25a6f9c9d9e4551a08 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Wed, 25 Sep 2024 06:50:00 +0000
Subject: [PATCH] data: install appdata file to correct location

Appdata files used to be installed to `/usr/share/appdata`. However,
this location has been changed to `/usr/share/metainfo` in recent
versions of the AppStream specification.
---
 data/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index e5319f5..650b5d0 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -29,7 +29,7 @@ appstream_file = i18n.merge_file(
   output: '@0@.appdata.xml'.format(app_id),
   po_dir: '../po',
   install: true,
-  install_dir: solanum_datadir / 'appdata'
+  install_dir: solanum_datadir / 'metainfo'
 )
 
 appstreamcli = find_program('appstreamcli', required: false)
-- 
GitLab

